includes/WikiPage.php

```diff
@@ -1554,6 +1560,12 @@ public function doEdit( $text, $summary, $flags = 0, $baseRevId = false, $user =
 		// Return the new revision (or null) to the caller
 		$status->value['revision'] = $revision;
 
+		$links2= null;
```

Comment:
Is that correct? Seems like $links2 is unused.

Reply:
to be deleted